.gitignore

```diff
@@ -168,3 +168,10 @@ models
 fe_files
 
 document.json
+
+
+# add output of image extractor and relevant input files
+fe-exams/
```

Comment:
Are the fe-exams that are version controlled supposed to be there?

Reply:
If you do want the pdfs version controlled, can you run the following and then make another commit:
```
git lfs track '*.pdf'
```